pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -52,7 +52,12 @@ func CreateDomainInterfaces(vmi *v1.VirtualMachineInstance, domain *api.Domain,
 			return nil, fmt.Errorf("failed to find network %s", iface.Name)
 		}
 
-		if iface.Binding != nil || iface.SRIOV != nil || iface.Slirp != nil {
+		var attachmentType string
+		if c.DomainAttachmentByInterfaceName != nil {
+			attachmentType = c.DomainAttachmentByInterfaceName[iface.Name]
+		}
```

Comment:
Found the second time it was used.
You probably do not need this.

Up to you.